src/features/input-editor/components/InputEditor/Table.jsx

```
@@ -1,62 +1,51 @@
-import {
-  useChanges,
-  useDiscardChanges,
   useSelected,
-  useSetSelected,
+  useSetSelectedFromTable,
 } from 'features/input-editor/stores/inputEditorStore';
 import ErrorBoundary from 'antd/es/alert/ErrorBoundary';
-import { useSetShowLoginModal } from 'features/auth/stores/login-modal';
+import { TableButtons } from 'features/input-editor/components/table-selection-buttons';
 
 const title = `You can select multiple buildings in the table and the map by holding down the "${getOperatingSystem() == 'Mac' ? 'Command' : 'Control'}" key`;
```

ESLint

[error] 'title' is assigned a value but never used.

(no-unused-vars)